### PR TITLE
pegasus: Add version alpha16-42-g996720eb

### LINKED ISSUE #518: Add Pegasus Frontend
[Pegasus](https://pegasus-frontend.org) is a frontend launcher for game emulators (such as those currently in bucket)
- Files are distributed via [GitHub](https://github.com/mmatyas/pegasus-frontend/releases)
- Placing "portable.txt" in the application root will cause it to run in portable mode ([docs](https://pegasus-frontend.org/docs/user-guide/portable/))
- Portable mode configuration files are located in "/config" in the applcation root ([docs](https://pegasus-frontend.org/docs/user-guide/config-dirs/))
- Possible manifest would create and persist the portable.txt and config folder

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #518

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/pegasus.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "alpha16-42-g996720eb",
+    "homepage": "https://pegasus-frontend.org",
+    "description": "A graphical frontend application for browsing your game library and launching all kinds of emulators from the same place.",
+    "license": "GPL-3.0-only",
+    "notes": "Appication settings are stored in ~\\AppData\\Local\\pegasus-frontend directory. Or the config folder inside the $app directory",
```
Comment (Calinou) [inline bucket/pegasus.json:6]:
```suggestion
    "notes": "Application settings are stored in the \"%LOCALAPPDATA%\\pegasus-frontend\" directory, or the \"config\" folder inside the \"$app\" directory.",

```

File: bucket/pegasus.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "alpha16-42-g996720eb",
+    "homepage": "https://pegasus-frontend.org",
+    "description": "A graphical frontend application for browsing your game library and launching all kinds of emulators from the same place.",
```
Comment (Calinou) [inline bucket/pegasus.json:4]:
```suggestion
    "description": "Graphical frontend for browsing your game library and launching all kinds of emulators from the same place",

```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by Zliced13** [comment]
Your welcome @Calinou

**Pull request merged**